content/docs/300-on-brand-menu-api/300-api-reference/100-get-menus.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Sub-section A
+title: GET /v1/menus
```

Comment:
Would be nice if we could format this as code instead of plain text - similar to how the page navigation works.
<img width="180" alt="image" src="https://user-images.githubusercontent.com/1712435/174564365-6af8dbbc-7e33-4321-8fe9-255c5b92b12d.png">

This is how it's currently rendering.
<img width="345" alt="image" src="https://user-images.githubusercontent.com/1712435/174564525-41bb790b-2adc-4e20-97c8-eb9dd73b0068.png">

Card to track this work (for a future release) here: https://trello.com/c/0iwwVNy5/4349-developer-docs-site-code-formatting-support-in-main-navigation

Reply:
I did try wrapping the title in back-ticks (\`GET /...\`), but that didn't work.

Reply:
Np! Happy to punt on this for v1